packages/expo-modules-autolinking/src/types.ts

```diff
@@ -311,9 +325,59 @@ export interface RawExpoModuleConfig {
    */
   devtools?: {
     /**
-     * The webpage root directory for Expo CLI DevTools to serve the web resources.
+     * The webpage root directory for Expo CLI DevTools to serve the web resources. Only set if the module has a web interface.
+     */
+    webpageRoot?: string;
+    /**
+     * Cli extension config for the module.
      */
-    webpageRoot: string;
+    cli?: {
+      /*
+       * The description of the module that will be displayed in the CLI.
+       */
+      description: string;
+      /**
+       * The commands that the module provides in the CLI.
+       * Each command has a name and a caption.
+       */
+      commands: {
+        /**
+         * Name of command
+         */
+        name: string;
+        /**
+         * caption for the command that will be displayed in the CLI.
+         */
+        caption: string;
+        /**
+         * Optional array of disabled environments for the command. By default all commands are enabled on all environments.
+         * Environments can be 'cli' for the CLI or 'mcp' for the Model Context Protocol.
+         */
+        disabled?: ('cli' | 'mcp')[];
```

Comment:
maybe remove this first that we can add mcp from a separate pr.
i was thinking maybe `environments: ('cli'| 'mcp')[]` would be clearer

Reply:
Good point! What I did was to rename it to `environments` and then type it as an array of either `mcp` or `cli` - that way we don't need a redundant declaration and can mark a command directly with either one or both.